### PR TITLE
[AR] Translation of 05.md

### PR DESCRIPTION
**Pull request by Alsanabani**
[AR] Translation of 05.md

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -353,6 +353,12 @@ ja:
   chapters:
     - path: ja/week07/07.md
       sections:
-        - path: ja/week07/07-1.md
+        - path: ja/week07/07-1.md<<<<<<< master
```
Comment (Atcold) [inline docs/_config.yml:356]: What is this `<<<<<<< master`?

File: docs/ar/week05/05.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!-----
+lang-ref: ch.05
+title: Week 5
+---
+-->
+
```
Comment (Atcold) [inline docs/ar/week05/05.md:6]: Remove this.

File: docs/ar/week05/05.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+---
+lang-ref: ch.05
+title: Week 5
```
Comment (Atcold) [inline docs/ar/week05/05.md:9]: Title translation is missing.

File: docs/ar/week05/05.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+<!--## Lecture part A
+-->
+
+## الجزء اﻷول من المحاضرة
```
Comment (Atcold) [inline docs/ar/week05/05.md:15]: Is it possible to use "A"?

File: docs/ar/week05/05.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+<!--We begin by introducing Gradient Descent. We discuss the intuition and also talk about how step sizes play an important role in reaching the solution. Then we move on to SGD and its performance in comparison to Full Batch GD. Finally we talk about Momentum Updates, specifically the two update rules, the intuition behind momentum and its effect on convergence.
+-->
+
+نبدأ بمقدمة عن الانحدار التدريجي. سنناقش الفكرة ونتحدث أيضًا عن كيف تلعب أحجام الخطوات دورًا مهمًا في الوصول إلى الحل. ثم ننتقل إلى النزول الاشتقاقي العشوائي (SGD - Stochastic Gradient Descent) وأدائها مقارنةً بـحزمة النزول الاشتقاقي الكامل (Full Batch GD). أخيرًا نتحدث عن تحديثات الزخم وبالتحديد قاعدتي التحديث، و الفكرة وراء الزخم، وتأثيره على التقارب.
+
+<!--
+## Lecture part B
+-->
+
+## الجزء الثاني من المحاضرة
```
Comment (Atcold) [inline docs/ar/week05/05.md:26]: Is it possible to use "B"?

File: docs/en/week05/05.md [hunk 6]
```diff
@@ -1,18 +1,43 @@
...
+-->
+
+## التدريب العملي
+
+<!--We briefly review the matrix-multiplications and then discuss the convolutions. Key point is we use kernels by stacking and shifting. We first understand the 1D convolution by hand, and then use PyTorch to learn the dimension of kernels and output width in 1D and 2D convolutions examples. Furthermore, we use PyTorch to learn about how automatic gradient works and custom-grads.
+-->
+
+نراجع بإيجاز مضاعفات المصفوفات ثم نناقش التلافيف. النقطة الأساسية هي أننا نستخدم الانوية (kernals) عن طريق التكديس والازاجة. نفهم أولاً الالتفاف ذو البعد الوحد (1D convolution) يدويًا ، ثم نستخدم PyTorch لمعرفة أبعاد النواة وعرض الإخراج في أمثلة التلافيف 1D و 2D. علاوة على ذلك ، نستخدم PyTorch للتعرف على كيفية عمل الانحدار التلقائي والتدرجات المخصصة.
 
-We briefly review the matrix-multiplications and then discuss the convolutions. Key point is we use kernels by stacking and shifting. We first understand the 1D convolution by hand, and then use PyTorch to learn the dimension of kernels and output width in 1D and 2D convolutions examples. Furthermore, we use PyTorch to learn about how automatic gradient works and custom-grads.
```
Comment (Atcold) [inline docs/en/week05/05.md:18]: Remove any edit to the English version.

File: docs/_config.yml (hunk 1 above)
Comment (Alsanabani) [reply docs/_config.yml]: Oops, this shouldn't be there.. 
Okay.. it's been removed

File: docs/ar/week05/05.md (hunk 2 above)
Comment (Alsanabani) [reply docs/ar/week05/05.md:6]: removed..

File: docs/ar/week05/05.md (hunk 3 above)
Comment (Alsanabani) [reply docs/ar/week05/05.md:9]: updated:
title: Week 5  -->  العنوان: الاسبوع 5

File: docs/ar/week05/05.md (hunk 4 above)
Comment (Alsanabani) [reply docs/ar/week05/05.md:15]: Yes it is possible, but ,if you don't mind, let me discuss this suggestion on Slack with the others in order to come out with standard formula in all chapters.

File: docs/en/week05/05.md (hunk 6 above)
Comment (Alsanabani) [reply docs/en/week05/05.md]: All edits are  removed and file returned as it was ..

File: docs/ar/week05/05.md (hunk 3 above)
Comment (Atcold) [reply docs/ar/week05/05.md:9]: The title is still in English. `lang: ar` is missing. `translator:` and `translation-date:` are missing too.

File: docs/ar/week05/05.md (hunk 2 above)
Comment (Atcold) [reply docs/ar/week05/05.md:6]: I can still see it.

Comment (Alsanabani) [reply docs/ar/week05/05.md]: I hope you don't see it now.

File: docs/ar/week05/05.md (hunk 3 above)
Comment (Alsanabani) [reply docs/ar/week05/05.md]: fixed and updated as requested

File: docs/ar/week05/05.md (hunk 4 above)
Comment (Alsanabani) [reply docs/ar/week05/05.md]: Finally, we got agreed to use : A المحاضرة

File: docs/ar/week05/05.md (hunk 5 above)
Comment (Alsanabani) [reply docs/ar/week05/05.md]: yes.. it is already updated --> : B المحاضرة

File: docs/ar/week05/05.md (hunk 4 above)
Comment (Alsanabani) [reply docs/ar/week05/05.md]: الجزء اﻷول من المحاضرة -- > المحاضرة أ
image

**Review by Atcold — APPROVED** [review]

**Pull request merged**